src/wp-includes/class-wp-script-modules.php

```diff
@@ -210,11 +210,26 @@ public function print_script_module_preloads() {
 	/**
 	 * Prints the import map using a script tag with a type="importmap" attribute.
 	 *
+	 * @global WP_Scripts $wp_scripts The WP_Scripts object for printing the polyfill.
 	 * @since 6.5.0
 	 */
 	public function print_import_map() {
 		$import_map = $this->get_import_map();
 		if ( ! empty( $import_map['imports'] ) ) {
+			global $wp_scripts;
+			if ( isset( $wp_scripts ) ) {
```

Comment:
What would be the case when `$wp_scripts` is not set? I'm asking because I don't see it guarded anywhere else in the codebase?